backend/runner/taskrun/executor.go

```diff
@@ -818,3 +825,297 @@ func getRepositoryAndVCS(ctx context.Context, storage *store.Store, repoUID, vcs
 	}
 	return repo, vcs, nil
 }
+
+type updateDatabaseConfigAction string
```

Comment:
Can we move this code to a different file in the package. We might end up relocating this code somewhere else eventually.

Reply:
Done, move to `database_config_merge.go`.